### PR TITLE
Document how to cite sources with special characters.

### PR DESCRIPTION
**Pull request by 01mf02**
I got tripped by citing sources with slashes in their names, such as appearing in DBLP entries.
@Enter-tainer helped me with this in #3251.
I updated the documentation to show how to handle such a case, for which I added a DBLP entry to the bibliography.

I ran `cargo test --workspace --test tests`, but I was not able to generate the documentation, so I would be grateful if you could show me how to do that.

### CODE REVIEW

File: crates/typst/src/model/cite.rs [hunk 1]
```diff
@@ -24,6 +24,12 @@ use crate::text::{Lang, Region, TextElem};
 /// You can also call `cite`
 /// explicitly. #cite(<arrgh>)
 ///
+/// If your source name contains
+/// certain characters such as slashes
+/// which are not recognized by the `<>` syntax,
+/// you can use `label` instead.
+/// #cite(label("DBLP:books/lib/Knuth86a"))
```
Comment (laurmaedje) [inline crates/typst/src/model/cite.rs:31]: Please move the explanation out of the example into actual prose. You can add a second example below the first one.

File: tests/ref/meta/bibliography-full.png
Comment (laurmaedje) [inline tests/ref/meta/bibliography-full.png:1]: Please increase the page size of bibliography-full so that everything fits onto one page.

Comment (01mf02) [reply tests/ref/meta/bibliography-full.png:1]: Done.

File: crates/typst/src/model/cite.rs (hunk 1 above)
Comment (01mf02) [reply crates/typst/src/model/cite.rs]: Done as well.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**